### PR TITLE
Added useVotingPowerBalanceList hook

### PR DESCRIPTION
**Pull request by ncomerci**
useVotingPowerBalanceList receives an array of addresses and returns the VP of each address

### CODE REVIEW

File: src/api/Snapshot.ts [hunk 1]
```diff
@@ -310,9 +311,17 @@ export class Snapshot extends API {
 
     for (const score of scores) {
       for (const address of Object.keys(score)) {
-        result[address.toLowerCase()] = (result[address.toLowerCase()] || 0) + Math.floor(score[address] || 0)
+        const addr = address.toLowerCase()
```
[the remark this answers is not in the document]
Comment (ncomerci) [reply src/api/Snapshot.ts:314]: Yes sorry, I forgot to change that line. thanks!

File: src/components/Section/ProposalVoting/VoteVotingPowerInfo.tsx [hunk 2]
```diff
@@ -10,13 +10,14 @@ import './DelegationsLabel.css'
...
-  vote: Vote | null
-  account: string | null
+  vote?: Vote | null
+  account?: string | null
 }
 
 const VoteVotingPowerInfo = ({ vote, account }: VotingSectionFooterProps) => {
-  const { votingPower, isLoadingVotingPower } = useVotingPowerBalance(account)
-  const hasEnoughVP = !!votingPower && votingPower > 0 && !isLoadingVotingPower
+  account = account?.toLowerCase()
```
Comment (andyesp) [inline src/components/Section/ProposalVoting/VoteVotingPowerInfo.tsx:18]: I'd avoid re-assigning props.

**Review by 1emu — APPROVED** [review]

**Comment by 1emu** [comment]
I like the idea of having just one method that knows how to execute the query depending on the amount of addresses received, but maybe we can call also have a hook that makes it easier to use if it's only one address:

```typescript
function useVotingPowerBalance(userAddress:string){
  const { votingPower, isLoadingVotingPower, hasEnoughToVote } = useVotingPowerBalances([userAddress])
  
  return {
    votingPower[userAddress],
    isLoadingVotingPower: state.loading,
    hasEnoughToVote[userAddress],
  }
}
```

**Review by andyesp — APPROVED** [review]

File: src/api/Snapshot.ts [hunk 3]
```diff
@@ -310,9 +311,16 @@ export class Snapshot extends API {
...
     for (const score of scores) {
       for (const address of Object.keys(score)) {
-        result[address.toLowerCase()] = (result[address.toLowerCase()] || 0) + Math.floor(score[address] || 0)
+        result[address] = (result[address] || 0) + Math.floor(score[address] || 0)
       }
     }
+
+    for (const addr of addresses) {
+      if (!result[addr]) {
+        result[addr] = 0
```
Comment (1emu) [inline src/api/Snapshot.ts:320]: If the idea of this is to initialize results for those addresses that aren't in score, I believe you can just move this up

```typescript
for (const addr of addresses) {
  result[addr] = 0
}

for (const score of scores) {
...
```

**Review by 1emu — APPROVED** [review]
LGTM!

**Pull request merged**